docs/css-style-guide.md

```diff
@@ -91,10 +98,9 @@ Where 'component' is a re-usable building block (eg. a tab bar),
 - Only reference classes with `js-` and `is-` prefixes in JavaScript code
 - Use names with a `js-` prefix for handle or "ref" classes that are used by JS to get a reference to particular DOM elements. These classes should not appear in stylesheets
 
-## SASS Features
+## Working in SASS
 
 - Use variables, functions and mixins
-- [Avoid @extends](https://www.sitepoint.com/avoid-sass-extend/). Use mixins instead
```

Comment:
We said this already

Reply:
I think it makes more sense to have the "avoid `@extends`" here rather than at the top though.